RELEASE_NOTES.md

```diff
@@ -85,6 +85,14 @@ making a backup is strongly advised.
   with different default settings, because input color profiles and 
   demosaicing need at least a rough white balancing early in the pipe.
 
+  Map view has evolved. Images close to each other are now grouped and a count of
```

Comment:
other -> others

Reply:
I've copied that from elstoc's text in dt doc, should be correct, shouldn't it  ?

Reply:
other is correct

Reply:
If elstoc wrote that it is correct no question about that! So I'm worng.